src/blr/cli/filterclusters.py

```diff
@@ -277,7 +277,7 @@ def write_molecule_stats(self, output_prefix, molecule_dict):
         # Writing molecule-dependant stats
         for barcode in tqdm(molecule_dict):
             number_of_molecules = len(molecule_dict[barcode])
-            molecules_per_bc.write(str(number_of_molecules))
+            molecules_per_bc.write(str(number_of_molecules) + "\n")
```

Comment:
This would be a good place to use `print` instead. Like this:
```
print(number_of_molecules, file=molecules_per_bc)
```
That way, you avoid the explicit `\n` and also `str()`.

Reply:
Cool, never used print for writing to files before.